odxtools/diaglayer.py

```diff
@@ -298,7 +299,7 @@ def ecu_variant_patterns(self) -> List[EcuVariantPattern]:
     # <stuff subject to value inheritance>
     #######
     @property
-    def diag_comms(self) -> NamedItemList[Union[DiagService, SingleEcuJob]]:
+    def diag_comms(self) -> NamedItemList[DiagComm]:
```

Comment:
breaking change?

Reply:
not really: The only two concrete subclasses of DIAG-COMM are DIAG-SERVICE and SINGLE-ECU-JOB...